src/line-graph.js

```diff
@@ -1,13 +1,34 @@
 import Highcharts from 'highcharts/highcharts';
-import {
-  AxisLeft
-} from './helpers/axisConverter';
-
+import { AxisLeft } from './helpers/axisConverter';
 export class LineGraph extends HTMLElement {
-  constructor() {
-    super();
-  }
-  /**
+
+    constructor() {
+        super();
+        this.height;
```

Comment:
Can hoist here in the constructor or elsewhere, it's technically not needed but may help readability, this can come in handy if we convert to Typescript. I'd also be open to these being a `getter` & `setter`.

Reply:
yeh! you are right, there would be a lot of properties as we will go ahead in high-chart developments.

Reply:
Okay @FarajiA